samples/_bootstrap.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+import re
+
+STORAGE_ACCOUNT_NAME = os.environ.get("AZSTORAGETORCH_STORAGE_ACCOUNT_NAME")
+CONTAINER_NAME = os.environ.get("AZSTORAGETORCH_CONTAINER_NAME")
+
+
+def modify_url(path):
+    if not STORAGE_ACCOUNT_NAME or not CONTAINER_NAME:
+        print(
+            "Please set environment variables AZSTORAGETORCH_STORAGE_ACCOUNT_NAME and AZSTORAGETORCH_CONTAINER_NAME"
+        )
+        return
+
+    with open(path, "r") as f:
+        content = f.read()
+
+    pattern = r'(CONTAINER_URL\s*=\s*\(\s*"https://)([^.]+)(\.blob\.core\.windows\.net/)([^"]+)("\s*\))'
+    new_url = rf"\1{STORAGE_ACCOUNT_NAME}\3{CONTAINER_NAME}\5"
+
+    return re.sub(pattern, new_url, content)
+
+
+def run_sample(path):
+    modified_content = modify_url(path)
+    if modified_content:
+        local_vars = {}
+        exec(modified_content, locals=local_vars)
```

Comment:
Let's avoid calling `exec()`. Instead let's just use subprocesses such as [`subprocess.run()`](https://docs.python.org/3/library/subprocess.html#subprocess.run) to invoke a python interpreter (e.g. by using `sys.executable` as the command to run). The main reasons that I'm suggesting this is running `exec` can lead to problematic side effects (e.g. mutating state in the current process) and should be avoided unless we really need to use it. In using subprocesses, the execution of the code is much more sandboxed to the child process and easier to manage.